src/AutoComplete/AutoComplete.component.tsx

```diff
@@ -125,9 +138,24 @@ export const AutoComplete = ({
     const [isFocused, setIsFocused] = useState<boolean>(false);
     // The current search term
     const [term, setTerm] = useState<string>(value ? `${value}` : '');
+    const autoFocusRef = useRef<HTMLDivElement>(null);
     // Instance of the nested input
     const inputRef = useRef<any>();
     const resultsRef = useRef<any>();
+    let rootElement: any;
+
+    // Handle click outside of auto complete component
+    const handleOutsideClick = (e: any) => {
+        if (isFocused === true && autoFocusRef.current && !autoFocusRef.current.contains(e.target)) {
```

Comment:
oh wait. I will update this line to just `isFocused` instead of the `=== true nonsense`